index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+<section class="Workssection1">
+    <header class="Header">
+        <a class="mylogo">MY logo</a>
+         <nav>
+            <a href="#"> </a>
+            
+                   </nav>
+                   <h class="burger">
+                    <div class="humbergericon"></div>
+                    <div class="humbergericon"></div>
+                    <div class="humbergericon"></div>  
+                    </h>
+                                 
+    </header>
+     <div class="cardBack01">
+        <div>
+            <h1 class="main-header"> I'm Natnael Glad to see you!</h1>
+            <p class="section1"> I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what 
+                you see and have a project you need coded, don’t hestiate to contact me. </p>
+        </div>
+        <p>
+        <div class = "social-section">
+                    <h4 class="connectmywork"> LET’S CONNECT</h4>
+                <ul class="social-icons-mywork">
+                    <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
+                    <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
+                </ul>
+        </div>
```

Comment:
- [ ] nice work 👍🏿 I see you are wrapping a `div` inside a paragraph tag which is not supposed to be please do well to revert and also please add a padding-bottom to the bottom it your icon also 😉 
 - [ ] please a an image attribute to your image tag